docs/content/direct/combined-status.md

```diff
@@ -8,38 +8,85 @@ It may make sense to expand the latter into a general feature in the future, acc
 
 The general technique involves the following ideas.
 
-1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a small subset of the expression language.
+1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a different expression language (one of growing currency in the Kubernetes milieu).
```

Comment:
Should we mention which "different expression language"  ?

Reply:
Yes, updated.